_projects/2024/100452420/100452420.Rmd

```diff
@@ -0,0 +1,364 @@
+---
+title: "Serena Williams Career"
+description: |
+  it shows the ranking of the famous tennis player.
+categories: "2024"
+author: Diego Fernández-Alvarez
+date:  "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(out.width="100%", fig.align="center", fig.showtext = TRUE)
+```
+
+This post details my process of replicating a chart that visualizes Serena Williams's world tennis ranking from 1998 to 2022. The original chart was published by The New York Times, and you can view it [here.](https://www.nytimes.com/2022/09/08/learning/whats-going-on-in-this-graph-sept-14-2022.html)
+
+
+Below is the original chart for reference:
+
+
+![Original Plot](../100452420/original_plot.png)
```

Comment:
Please set it as "external" as e.g. David does [here](https://github.com/IBiDat/dataviz/blob/81084e40371bd8161998806e9c7471e04ad47cfe/_projects/2024/100535712/100535712.Rmd#L26).